distributed/driver/driver_test.go

```diff
@@ -0,0 +1,40 @@
+package driver
+
+import (
+	"context"
+	"net"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/chrislusf/gleam/flow"
+	"google.golang.org/grpc"
+)
+
+func TestConcurrentDriver(t *testing.T) {
+	srv := grpc.NewServer()
```

Comment:
I do not understand this section. The server and listener are created but not used.

Reply:
yeah, and still the test crashes regularly due to data races